docs/api/structures/offscreen-shared-texture.md

```diff
@@ -0,0 +1,23 @@
+# OffscreenSharedTexture Object
```

Comment:
I think we'd need to see if our Docs Parser code supports this, but I wonder if it'd be possible to mark this Object as experimental as well since it's only used in conjunction with experimental APIs.

Reply:
We could just test out the syntax and see if the Artifact Comparison violently fails or adds the proper `@experimental` TSDoc tag: https://github.com/electron/electron/pull/42953/checks?check_run_id=28222213375

Reply:
I checked the generated electron.d.ts and it seems good? Ok, I'll add experimental.

Reply:
Seems not able to add  _Experimental_ behind or before `Object`.

Reply:
That seems fine to me :)